ludwig/features/numerical_feature.py

```diff
@@ -327,10 +326,14 @@ def inverse_transform(self, x: np.ndarray) -> np.ndarray:
         return x * self.sigma + self.mu
 
     @staticmethod
-    def fit_transform_params(column: np.ndarray) -> dict:
+    def fit_transform_params(
+            column: np.ndarray,
+            backend: 'LocalBackend'
```

Comment:
The type here can be `Backend` (the base class).

Reply:
done